docs/content/direct/commission-workloads.md

```diff
@@ -0,0 +1,5 @@
+# Deploying Workloads KubeStellar Usage Examples
```

Comment:
We still need work on concepts and terminology. The name of this file is "commission". The title says "Deploying". The first sentence speaks of "defining, binding, and deploying". I am not clear on what is meant by "Commission" and by "deploy".

I think of "deploy" as something that happens in stages, by virtue of action of different agents. A user of KubeStellar _defines_ their workloads in WDSes, and also creates in the WDSes control objects that bind workload(s) with WEC(s). KubeStellar and OCM transform and transport the workload(s) from WDS(es) to WEC(s).

Reply:
OK, your sense of _deploy_ is what I meant by _commission_

To me, _deploy_ refers to the final delivery and start of the workload on the WEC. Hence our mutual confusion here.

Reply:
I have updated the title to match the menu, recognizing that we will revisit the terminology in a future PR